src/main/drivers/light_ws2811strip_hal.c

```diff
@@ -86,7 +86,7 @@ void ws2811LedStripHardwareInit(ioTag_t ioTag)
 
     ws2811IO = IOGetByTag(ioTag);
     IOInit(ws2811IO, OWNER_LED_STRIP, 0);
-    IOConfigGPIOAF(ws2811IO, IO_CONFIG(GPIO_MODE_AF_PP, GPIO_SPEED_FREQ_VERY_HIGH, GPIO_PULLUP), timerHardware->alternateFunction);
+    IOConfigGPIOAF(ws2811IO, IO_CONFIG(GPIO_MODE_AF_PP, GPIO_SPEED_FREQ_VERY_HIGH, GPIO_PULLDOWN), timerHardware->alternateFunction);
```

Comment:
Isn't the just `GPIO_NOPULL`? There is no need to pull up or down for LED strip, I think.

Reply:
It shouldn't matter as long as pin is configured as output (always for ledstrip). But it is safer to pull some where (up or down) to prevent floating pin when code is changed